po/pt_BR.po

```diff
@@ -7,117 +7,154 @@ msgid ""
 msgid "The first feature update to extension-manager. Highlights include:"
 msgstr ""
 "A primeira atualização de funções do gerenciador de extensões. Em destaque:"
 
-#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:50
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:77
 msgid "Dark theme and support for overriding the system colour scheme"
-msgstr "Tema escuro e suporte para sobrescrever o esquema de cores do sistema"
+msgstr "Tema escuro e suporte para substituir o esquema de cores do sistema"
 
-#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:51
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:78
 msgid "Shows screenshots in extension listings"
 msgstr "Exibe capturas de tela nos detalhes das extensões"
 
-#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:52
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:79
 msgid "Shell version compatibility check"
 msgstr "Verifica a compatibilidade de versão do Shell"
 
-#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:53
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:80
 msgid "Localised into seven languages"
-msgstr "Traduzido para sete idiomas"
+msgstr "Traduzido em sete idiomas"
 
-#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:54
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:81
 msgid "Sort search results"
-msgstr "Resultado de busca ordenado"
+msgstr "Resultados de busca ordenado"
 
-#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:55
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:82
 msgid "Global extension on/off toggle"
 msgstr "Ativar/desativar extensão global"
 
-#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:56
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:83
 msgid "Displays user and system extensions separately"
 msgstr "Exibe as extensões do usuário e do sistema separadamente"
 
-#: data/com.mattjakeman.ExtensionManager.appdata.xml.in:57
-msgid "Performance and stability"
-msgstr "Desempenho e estabilidade"
-
-#: data/com.mattjakeman.ExtensionManager.gschema.xml:11
-msgid "Colour Scheme Preference"
-msgstr "Preferência do esquema de cores"
+#: src/exm-application.c:106
+msgid "Extension Manager (Development)"
+msgstr "Gerenciador de Extensões (Desenvolvimento)"
 
-#: data/com.mattjakeman.ExtensionManager.gschema.xml:12
-msgid ""
-"Whether to follow the system colour scheme or force either light or dark "
-"mode."
-msgstr "Seguir as cores do sistema ou forçar o modo claro ou escuro."
+#: src/exm-application.c:107
+msgid "Extension Manager"
+msgstr "Gerenciador de Extensões"
 
 #. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
-#: src/exm-application.c:112
+#: src/exm-application.c:117
 msgid "translator-credits"
 msgstr ""
-"Douglas Silva <douglasilvadev@gmail.com>"
-"Tiago Lucas Flach https://github.com/tiagoFlach"
+"Douglas Silva <douglasilvadev@gmail.com>Tiago Lucas Flach https://github."
```

Comment:
I also missed this before, but there should be a newline `\n` here otherwise both your names will appear as one entry.


Like this:
```
"Douglas Silva <douglasilvadev@gmail.com>\n"
"Tiago Lucas Flach https://github.com/tiagoFlach"
```

Reply:
Yeah, it need a line break right there. Thanks a lot!